installer/docker/ubuntu/jdk/build.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script to create a Docker image using Ubuntu and latest AdoptOpenJDK  linux binary
+# This script works in conjunction with jenkins plugins
+# The script assumes that Jenkins has placesd the tar.gz of the binary
+# in the root of the workspace. (WORKSPACE envar is set by jenkins)
+#
+# REPO defines the target docker repository for tagging images with
+# ARCH defines the computer architecture tag to be used for docker image tags
+# that are machine specific.
+#
+# This script builds the docker image but does not push to docker hub.
+# The docker plugin could build the image but is not dynamically configurable
+# so is only used to push the created image.
+#
+
+: "${WORKSPACE:=$PWD}"
+: "${REPO:=adoptopenjdk/openjdk8}"
+: "${ARCH:=x86_64}"
+
+# remove any locally built images before building
+
+images=$(docker images | grep "^$REPO"  | awk '{print $3}' | uniq)
+
+for i in $images
+do
+   docker rmi -f "$i"
+done
+
+# create place to hold image to be dockerized
+rm -rf docker_image
+mkdir -p docker_image
+
+# add contents to image
+cp Dockerfile docker_image
+cd docker_image || exit
+mv "$WORKSPACE"/OpenJDK8*.tar.gz .
```

Comment:
Well my assumption is that this particular process is for 8 only. Docker runtime images for 9+ will be generated differently due to the new wonders of the module system.